pkg/check/info.go

```diff
@@ -37,7 +38,8 @@ func CollectClusterInfo(ctx *CheckContext) error {
 func collectVCenterInfo(ctx *CheckContext) {
 	version := ctx.VMClient.ServiceContent.About.Version
 	apiVersion := ctx.VMClient.ServiceContent.About.ApiVersion
+	build := ctx.VMClient.ServiceContent.About.Build
 	uuid := ctx.VMClient.ServiceContent.About.InstanceUuid
 	ctx.ClusterInfo.SetVCenterVersion(version, apiVersion)
-	vCenterInfoMetric.WithLabelValues(version, apiVersion, uuid).Set(1.0)
+	vCenterInfoMetric.WithLabelValues(version, apiVersion, uuid, build).Set(1.0)
```

Comment:
btw IIRC build number is not available on older versions such as 6.7u3. But those are not supported anymore right? So we should be okay?

Reply:
> btw IIRC build number is not available on older versions such as 6.7u3
6.7 API doc: https://developer.vmware.com/apis/358/vsphere/doc/vim.AboutInfo.html